calibration_utils.py

```diff
@@ -395,13 +395,32 @@ def calibrate_extrinsics(self, images_left, images_right, M_l, d_l, M_r, d_r, gu
         self.objpoints = []  # 3d point in real world space
         self.imgpoints_l = []  # 2d points in image plane.
         self.imgpoints_r = []  # 2d points in image plane.
-
-        images_left = glob.glob(images_left + "/*")
-        images_right = glob.glob(images_right + "/*")
-
+        left_path = images_left
+        right_path = images_right
```

Comment:
Why not change the `images_left` and `images_right` to left_path and left_right in the function call here `def calibrate_extrinsics(self, images_left, images_right, M_l, d_l, M_r, d_r, guess_translation, guess_rotation):`